lib/doekbase/data_api/tests/examples/get_mRNAs_for_gene_to_GFF.py

```diff
@@ -1,18 +1,17 @@
 import doekbase.data_api
 from doekbase.data_api.taxonomy.taxon.api import TaxonAPI
-from doekbase.data_api.annotation.genome_annotation import GenomeAnnotationAPI
+#from doekbase.data_api.annotation.genome_annotation import GenomeAnnotationAPI
+from doekbase.data_api.annotation.genome_annotation.api import GenomeAnnotationAPI
 from doekbase.data_api.sequence.assembly.api import AssemblyAPI
 from doekbase.data_api.core import ObjectAPI
 import time
+import os
 
 services = {"workspace_service_url": "https://ci.kbase.us/services/ws/",
             "shock_service_url": "https://ci.kbase.us/services/shock-api/"}
 
 
-import os
-token = os.environ["KB_AUTH_TOKEN"]
-
-def get_gff(gene):
+def get_gff(gene, genome_annotation):
```

Comment:
Ok, but here we run into the issue that we need a run() method or
equivalent for the test runner. This gets back to the testable nature of
examples. The get_gff content could be in the main body but that doesn't
really achieve anything.

On Mon, Jan 11, 2016 at 3:44 AM, Matt Henderson <notifications@github.com>
wrote:

> In lib/doekbase/data_api/tests/examples/get_mRNAs_for_gene_to_GFF.py
> <https://github.com/kbase/data_api/pull/149#discussion_r49312563>:
>
> >
> >  services = {"workspace_service_url": "https://ci.kbase.us/services/ws/",
> >              "shock_service_url": "https://ci.kbase.us/services/shock-api/"}
> >
> >
> > -import os
> > -token = os.environ["KB_AUTH_TOKEN"]
> > -
> > -def get_gff(gene):
> > +def get_gff(gene, genome_annotation):
>
> It seems like you don't really need the function get_gff() in
> get_mRNAs_for_gene_to_GFF.py
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kbase/data_api/pull/149/files#r49312563>.
>